flow.js

```diff
@@ -97,9 +97,13 @@ class Flow {
         {condition: 'electricsLightsSwitches', nextStep: 'repair-bedroom-lighting-problems'},
         {condition: 'wallsFloorsCeiling', nextStep: 'wall-floor-ceiling-problems'},
         {condition: 'windows', nextStep: 'repair-window-problems'},
-        {condition: 'damagedOrStuckDoors', nextStep: 'repair-door-problems'},
+        {condition: 'damagedOrStuckDoors', nextStep: 'repair-living-area-door-problems'},
         {condition: 'dampOrMould', nextStep: 'damp-mould-problems'}]
       },
+      'repair-living-area-door-problems': { prevSteps: '', nextStep: [
+        {condition: 'internalDoorIssue', nextStep: 'repair-description'},
+        {condition: 'lockOnDoor', nextStep: 'not-eligible-non-emergency'},
```

Comment:
Selecting `lockOnDoor` should lead to `repair-description`, not `not-eligible-non-emergency`

Reply:
[This](https://trello.com/c/IL2zJQ78/196-identify-repair-door-living-areas-bathroom-bedroom) is the conversation that I had with Fraser

Reply:
The above is the correct behaviour and is confirmed in the ticket linked in the above [comment](https://github.com/City-of-Lincoln-Council/housing-repairs-online-frontend/pull/72#discussion_r798602353).